Formula/z.lua.rb

```diff
@@ -0,0 +1,22 @@
+class ZLua < Formula
+  desc "New cd command that helps you navigate faster by learning your habits"
+  homepage "https://github.com/skywind3000/z.lua"
+  url "https://github.com/skywind3000/z.lua/archive/1.8.4.tar.gz"
+  sha256 "ae26580b5e0a3b3d82a932804e0d2cd9ead44cbb49edb3360d3c00d4ae12f06b"
+  head "https://github.com/skywind3000/z.lua.git"
+
+  bottle :unneeded
+
+  depends_on "lua"
+
+  def install
+    pkgshare.install "z.lua", "z.lua.plugin.zsh", "init.fish"
+    libexec.install "test_path.lua"
+    doc.install "README.md", "LICENSE"
+  end
+
+  test do
+    ENV["LUA_PATH"] = "#{pkgshare}/z.lua"
+    system "lua", "#{libexec}/test_path.lua"
```

Comment:
This seems to run a test file rather than testing a common user action, which is the type of test we require for homebrew. Maybe you can create a directory and cd into it?